### ISSUE TITLE
Training details

### ISSUE THREAD

**Issue by Tianhao-Qi** [Vchitect/SEINE#24]
Thanks for your interesting work!
Can you tell me the batch size, learning rate and iterations during training?

**Comment by chaidosa** [comment]
@xinyuanc91, could you provide the training details of the model like, dataloader, batch size, learning rate and iterations during training?
